ci-operator/step-registry/layered-product/opp/add-storage/layered-product-opp-add-storage-commands.sh

```diff
@@ -0,0 +1,38 @@
+#!/bin/bash
+set -x
+set -o nounset
+set -o errexit
+set -o pipefail
+
+export KUBECONFIG=${SHARED_DIR}/kubeconfig
+
+# create the policies namespace
+oc apply -f - <<EOF
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: openshift-storage
+EOF
+
+oc label namespace openshift-storage openshift.io/cluster-monitoring=true
+
+# create 3 machinesets for ocp storage on aws
+CLUSTERID=$(oc get machineset -n openshift-machine-api -o jsonpath='{.items[0].metadata.labels.machine\.openshift\.io/cluster-api-cluster}')
+echo $CLUSTERID
+
+curl -s https://raw.githubusercontent.com/red-hat-storage/ocs-training/master/training/modules/ocs4/attachments/cluster-workerocs-us-east-2.yaml | sed -e "s/CLUSTERID/${CLUSTERID}/g" | oc apply -f -
+
+
+# wait for storage nodes to be ready
+RETRIES=30
+for i in $(seq "${RETRIES}"); do
+  if [[ $(oc get nodes -l cluster.ocs.openshift.io/openshift-storage= | grep Ready) ]]; then
+    echo "storage worker nodes are up is Running"
+    break
+  else
+    echo "Try ${i}/${RETRIES}: Storage nodes are not ready yet. Checking again in 30 seconds"
+    sleep 30
```

Comment:
I think this is probably causing the reason we aren't seeing full compliance the timeout is 15 minutes...

I don't think this is enough we can increase this sleep to 60 to make it wait 30 minutes for compliance.